main.tf

```diff
@@ -96,6 +96,7 @@ resource "yandex_kubernetes_cluster" "kube_cluster" {
       kube_apiserver_enabled     = var.master_logging.enabled_kube_apiserver
       cluster_autoscaler_enabled = var.master_logging.enabled_autoscaler
       events_enabled             = var.master_logging.enabled_events
+      audit_enabled              = var.master_logging.audit_enabled
```

Comment:
Please rename the option according to other logging options naming
var.master_logging.audit_enabled -> enabled_audit

Reply:
Hi, done